### ISSUE TITLE
Fieldset border has insufficient contrast

### ISSUE THREAD

**Issue by msayoung** [localgovdrupal/localgov_base#757]
The fieldset border defaults to #b1b4b6

On a #ffffff background this equates to a 2.08:1 colour contrast. 

WCAG 1.4.11: Non-text Contrast say: 

The visual presentation of the following have a contrast ratio of at least 3:1 against adjacent color(s): 

User Interface Components: Visual information required to identify user interface components and states, except for inactive components or where the appearance of the component is determined by the user agent and not modified by the author;

https://www.w3.org/WAI/WCAG21/Understanding/non-text-contrast.html 

--
Example
https://site3.microsites.localgovdrupal.org/events 
https://demo.localgovdrupal.org/events

--- 
Recommendation - set the default to a colour with at least 3:1 

`   --color-grey-dark: #505a5f; ` has a contrast of 7:1 with white.  

thanks to @Mayur2609 for flagging this.
